library/src/include/rocblas_unique_ptr.hpp

```diff
@@ -1,6 +1,7 @@
 #ifndef GUARD_ROCBLAS_MANAGE_PTR_HPP
 #define GUARD_ROCBLAS_MANAGE_PTR_HPP
 
+#include "definitions.h"
```

Comment:
`rocblas_unique_ptr.hpp` will be removed with the device memory allocation refactoring, currently in-progress.

We do not need to use `unique_ptr` wrappers for `hipMalloc` and `hipFree` anymore, since we fully manage device memory inside of `rocblas_handle` now.

Reply:
Hi @leekillough ,

When can we get new device memory management??
We can modify this part for new mechanism.

Thanks,
Tony